buildscripts/Build-NetCore.ps1

```diff
@@ -0,0 +1,34 @@
+$dnxHome = Join-Path (Get-Location) ".dnx"
+$dnxPath = Join-Path $dnxHome "bin"
+
+Write-Host "Downloading dnvm to: $dnxPath"
+if (!(Test-Path $dnxPath)) { md $dnxPath | Out-Null }
+
+$dnvmPs1Path = Join-Path $dnxPath "dnvm.ps1"
+$dnvmCmdPath = Join-Path $dnxPath "dnvm.cmd"
+
+$wc = New-Object System.Net.WebClient
+$wc.DownloadFile('https://raw.githubusercontent.com/aspnet/Home/dev/dnvm.ps1', $dnvmPs1Path)
+$wc.DownloadFile('https://raw.githubusercontent.com/aspnet/Home/dev/dnvm.cmd', $dnvmCmdPath)
+
+Write-Host "Downloading dnx"
+
+$env:PATH = ("$dnxPath;" + $env:PATH)
+
+dnvm upgrade -r coreclr
```

Comment:
This is different to what I threw into TeamCity, if a new version gets released won't the new one get downloaded and the line below which has a hardcoded path will no longer work? I assume the new DNX doesn't work like beta8 where I just installed it into the same directory as dnvm.